manifests/1.2.0/opensearch-1.2.0.yml

```diff
@@ -64,3 +64,23 @@ components:
     platforms:
       - darwin
       - linux
+  - name: sql
+    repository: https://github.com/opensearch-project/sql.git
+    ref: "main"
+    checks:
+      - gradle:properties:version
+      - gradle:dependencies:opensearch.version: plugin
+  - name: dashboards-reports
+    repository: https://github.com/opensearch-project/dashboards-reports.git
+    ref: "main"
+    working_directory: "reports-scheduler"
+    checks:
+      - gradle:properties:version
+      - gradle:dependencies:opensearch.version
+  - name: opensearch-observability
+    repository: https://github.com/opensearch-project/trace-analytics.git
+    ref: "main"
+    working_directory: "opensearch-observability"
+    checks:
+      - gradle:properties:version
+      - gradle:dependencies:opensearch.version
```

Comment:
Looks like we have new plugin. Adding @peternied as he is looking over 1.2 release.

Reply:
Thanks for watching out for this, the repo for trace analytics has been included in the release process as https://github.com/opensearch-project/trace-analytics/issues/118.  To what is included or not is up to the repo to manage as they are onboard the release process 👍 .